### DISCUSSION TITLE
Is stable release will be the 10th beta release (`v1.0 == v0.10`)?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [typst/typst#1878] [Q&A] [answered]
Some projects release stable versions after 9th beta version, but some can have 10th or 100th of beta versions (`v0.123`). In which category Typst falls? I.e., is the last beta version number is already "defined" or it can be whatever ("we'll get there, when we'll get there")?

**Comment by laurmaedje** [comment, marked as answer]
We'll get there when we get there, but probably not by 0.10.

**Reply by Andrew15-5** [reply]
Great. A lot of room for improvement, new features.
